vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -139,15 +139,19 @@ func (a *APIInstaller) getResourceKind(path string, storage rest.Storage) (schem
 	}
 
 	object := storage.New()
-	fqKinds, _, err := a.group.Typer.ObjectKinds(object)
+	fqKinds, unversioned, err := a.group.Typer.ObjectKinds(object)
 	if err != nil {
 		return schema.GroupVersionKind{}, err
 	}
 
 	// a given go type can have multiple potential fully qualified kinds.  Find the one that corresponds with the group
 	// we're trying to register here
 	fqKindToRegister := schema.GroupVersionKind{}
+	unversionedCandidate := schema.GroupVersionKind{}
 	for _, fqKind := range fqKinds {
+		if unversioned && fqKind.Group == "" {
```

Comment:
`metav1.Status` is only registered as unversioned into the core v1 group. Our build apigroup webhook storage returns `Status` in the `New()` method (https://github.com/openshift/origin/blob/34fd627194cf6b6e428641c3b0abceeb8938adf7/pkg/util/rest/webhook.go#L56-L56). This code here checks on api installation time that the registered group matches to the storage being installed.

Hence, we have two choice: 1) we can register `Status` under the build apigroup (https://github.com/sttts/origin/pull/51/commits/adc5280a3b1ea58bf3738f8fc12e621399402a3c) or 2) make the api installer happy with unversioned kinds as well.

We chose (2) here to avoid side-effects from `Status` being registered under another apigroup.